---
title: Perturbational phenotyping of human blood cells reveals genetically determined
  latent traits associated with subsets of common diseases
authors:
- Max Homilius#
- Wandi Zhu#
- Samuel S Eddy
- Patrick C Thompson
- Huahua Zheng
- Caleb N Warren
- Chiara G Evans
- David D Kim
- Lucius L Xuan
- Cissy Nsubuga
- Zachary Strecker
- Christopher J Pettit
- Jungwoo Cho
- Mikayla N Howie
- Alexandra S Thaler
- Evan Wilson
- Bruce Wollison
- Courtney Smith
- Julia B Nascimben
- Diana N Nascimben
- Gabriella M Lunati
- Hassan C Folks
- Matthew Cupelo
- Suriya Sridaran
- Carolyn Rheinstein
- Taylor Mcclennen
- Shinichi Goto
- James G Truslow
- Sara Vandenwijngaert
- Calum A Macrae#
- Rahul C Deo#
date: '2023-01-01'
publishDate: '2024-09-19T01:09:21.595224Z'
publication_types:
- article-journal
publication: '*Nature Genetics* #co-corresponding authors'
doi: 10.1038/s41588-023-01600-x
abstract: Although genome-wide association studies (GWAS) have successfully linked
  genetic risk loci to various disorders, identifying underlying cellular biological
  mechanisms remains challenging due to the complex nature of common diseases. We
  established a framework using human peripheral blood cells, physical, chemical and
  pharmacological perturbations, and flow cytometry-based functional readouts to reveal
  latent cellular processes and performed GWAS based on these evoked traits in up
  to 2,600 individuals. We identified 119 genomic loci implicating 96 genes associated
  with these cellular responses and discovered associations between evoked blood phenotypes
  and subsets of common diseases. We found a population of pro-inflammatory anti-apoptotic
  neutrophils prevalent in individuals with specific subsets of cardiometabolic disease.
  Multigenic models based on this trait predicted the risk of developing chronic kidney
  disease in type 2 diabetes patients. By expanding the phenotypic space for human
  genetic studies, we could identify variants associated with large effect response
  differences, stratify patients and efficiently characterize the underlying biology.
  Precision   medicine strives to reclassify complex heterogeneous diseases into
  distinct biologically defined groups, thereby enabling targeted therapies and improved
  outcomes. Examples include the subdivision of common cancers by somatic driver mutations
  1 , the discovery of eosinophilic variants of asthma 2 and the recognition that
  some presentations of heart failure may arise from the accumulation of amyloidogenic
  proteins, which can be subdivided further based on the aggregating protein 3. The
  realization of precision medicine has
links:
- name: URL
  url: https://doi.org/10.1038/s41588-023-01600-x
---
